source/repository/scylladb/migrate/migration.go

```diff
@@ -0,0 +1,54 @@
+package migrate
+
+import (
+	"errors"
+	"fmt"
+	"github.com/golang-migrate/migrate/v4"
+	// Using a blank import for the Cassandra database package ensures that
+	// its side effects, such as registering the database driver, take place
+	// during program initialization. This is done without directly
+	// referencing any of its identifiers in the code.
+	_ "github.com/golang-migrate/migrate/v4/database/cassandra"
+	"github.com/golang-migrate/migrate/v4/source/iofs"
+	"os"
+)
+
+type Manager struct {
+	dir      string
+	host     string
+	keyspace string
+}
+
+const MigrationDBURL = "cassandra://%s/%s"
+
+func New(dir, host, keyspace string) *Manager {
+	return &Manager{
+		dir:      dir,
+		host:     host,
+		keyspace: keyspace,
+	}
+}
+
+func (m *Manager) Run() error {
+	sourceDriver, err := iofs.New(os.DirFS(m.dir), "migrations")
```

Comment:
If we can `embed` migration folder to binary please add `embed` statement to migration code.